index.html

```diff
@@ -1,29 +1,136 @@
 <!DOCTYPE html>
 <html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
+<head> 
+   <!--This section creates the title of the HTML document and links it the CSS file -->
+    <meta charset = "utf-8">
+    <title> Pepper Pete's Website</title>
+    <link rel="stylesheet" href="style.css">
+</head>
 
-    <!-- Set the charset we're using to UTF-8 -->
-    <meta charset="UTF-8" />
+<body>  <!--This section outlines the header logo, title and header list -->
+   <header> <div class="head"> 
+      <img width="80" src="logo_v3.png" alt="header logo">
+      <h1 style="color:aliceblue; background-color: #E0643C; margin-right: auto"> PEPPER PETE'S</h1> 
+   
+   <!--these are the menu options -->
+   <ul class="nav-list"> 
+      <li> <a href="Home.html"> Home </a></li>
+      <li> <a href="Menu.html"> Our Menu </a></li>
+      <li> <a href="About_Us.html"> About Us </a></li>
+      <li> <a href="Location.html"> Locations </a></li>
+      <li> <a href="Events.html"> Events </a></li>
+      <li> <a href="Nutrition.html"> Nutrition </a></li>
+   </ul>
+</div> </header>
 
-    <!-- Set the screens default zoom / scale to 1 -->
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+<!-- Displays the image and text for the hero section, in this case it's our new stir fry -->
+<div class = stir>  
```

Comment:
This is technically fine but I would have loved to have seen quotation marks around class names 

```html
<div class="stir"
   ...
</div>
```